### PR TITLE
Feature/skatch 12

### PR DESCRIPTION
**Pull request by tjdgns5272**
Change login and signup components with react-semantic-ui

### CODE REVIEW

**Review by tjdgns5272 — COMMENTED** [review]
LGTM^^

File: src/components/login/login.jsx [hunk 1]
```diff
@@ -1,46 +1,58 @@
...
-import styles from './login.module.css';
 
 const Login = () => {
   const history = useHistory();
-  const inputRef = useRef();
 
-  const handleLogin = (event) => {
-    event.preventDefault();
+  const goToPlay = (event) => {
+    console.log(event);
```
Comment (Hank-Choi) [inline src/components/login/login.jsx:11]: https://github.com/airbnb/javascript/tree/master/react

File: src/components/login/login.jsx [hunk 2]
```diff
@@ -1,46 +1,58 @@
...
     history.push('/user/login/set');
   };
-  const goToSignup = () => {
+  const goToSignUp = (event) => {
+    console.log(event);
     history.push('/signup');
   };
```
Comment (Hank-Choi) [inline src/components/login/login.jsx:17]: https://github.com/airbnb/javascript/tree/master/react

**Review by Hank-Choi — COMMENTED** [review]
나머지는 지금껏 했던 리뷰 보면서 고치기바람

**Pull request merged**